ar_transaction_changes.gemspec

```diff
@@ -2,6 +2,7 @@
 lib = File.expand_path('../lib', __FILE__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require 'ar_transaction_changes/version'
+require 'all_changed_attributes/version'
```

Comment:
Only because I was following your example: this is my first week at Shopify outside of onboarding, and I have no previous ruby experience.  I'll remove it.